lib/project_types/theme/messages/messages.rb

```diff
@@ -233,6 +235,8 @@ module Messages
           HELP
           select: "Select theme to delete",
           done: "%s theme(s) deleted",
+          no_themes_error: "No themes to delete.",
+          no_themes_resolution: "Do you have any non-live themes?",
```

Comment:
```suggestion
          no_themes_error: "You don't have any theme to be deleted.",
          no_themes_resolution: "Try to create an unpublished theme with {{command:theme push -u -t <theme_name>}}.",
```

Reply:
This looks like the same `no_themes_resolution` message as the `publish` one?

Reply:
This is the same message as above – do we want to specify that you need an unpublished theme to `delete`?

Reply:
Considering the other theme types (development/demo/live) and their purposes, I believe it's a fair suggestion. So, developers may be aware they can manage (create/delete) unpublished themes.